src/datasources/targeted-messaging/targeted-messaging.datasource.ts

```diff
@@ -72,15 +92,27 @@ export class TargetedMessagingDatasource
         updated_at: targetedSafe.updated_at,
       }));
     });
+
+    await this.cacheService.deleteByKey(
+      CacheRouter.getTargetedSafeCacheKey({
+        outreachId: createTargetedSafesDto.outreachId,
+      }),
```

Comment:
Addressing my initial nit re. the argument would mean this needs changing as well.

```suggestion
    await this.cacheService.deleteByKey(
      CacheRouter.getTargetedSafeCacheKey(createTargetedSafesDto.outreachId),
    );
```

Reply:
[ce9f899](https://github.com/safe-global/safe-client-gateway/pull/2000/commits/ce9f89920db5be3645b0c7dd3d38c847e2a34800)